bandit.yml

```diff
@@ -0,0 +1,3 @@
+---
+assert_used:
```

Comment:
I suggest to rename the `bandit.yml` to `.bandit.yaml` for consistency reasons with the rest of the config files.

I'd also put a link to `assert_used` in the file so it is easier to understand where this file belongs to and where to find other configuration options that you might want to look into.

```suggestion
# https://bandit.readthedocs.io/en/latest/plugins/b101_assert_used.html?highlight=assert_used
assert_used:
```